documentation/asciidoc/topics/proc_encrypting_endpoints_custom.adoc

```diff
@@ -0,0 +1,76 @@
+[id='using_custom_tls-{context}']
+= Using Custom Certificates and Keystores
+Add a PKCS12 keystore or TLS certificate/key pair to a secret and configure {brandname} to use it for encrypting endpoint traffic.
+
+.Procedure
+//Community
+ifndef::productized[]
+. Apply `tls_secret.yaml`.
++
+[source,options="nowrap",subs=attributes+]
+----
+include::cmd_examples/oc_apply_tls_secret.adoc[]
+----
++
+. Configure {brandname} pods to use the secret.
```

Comment:
Can we show and explain the cr_tls_custom.yaml as in the productized version?

Reply:
@rigazilla I've pushed changes that align the docs between community and product. tbh, it's better in my opinion to keep both community and product very similar. part of the purpose to source docu upstream is to capture community feedback. 

for community I added steps to explain how to use the default CRs from the repository.